src/screens/mpesaToCusd/SelectPaymentMethodScreen.tsx

```diff
@@ -0,0 +1,195 @@
+      </View>
+    </TouchableOpacity>
+  );
+};
+
+const BannerContent = (props) => {
+  return (
+    <View style={bannerStyles.container}>
+      <Image source={COMING_SOON} style={bannerStyles.image} />
+      <Text style={bannerStyles.title}>Coming soon</Text>
+      <Text style={bannerStyles.text}>
+        Stay put, we are soon adding cash option.
+      </Text>
+
+      <TouchableOpacity
+        style={bannerStyles.buttonContainer}
+        onPress={() => props.bannerRef.current?.closeBanner()}
+      >
+        <Text style={bannerStyles.button}>Dismiss</Text>
+      </TouchableOpacity>
+    </View>
+  );
+};
+
+const SelectPaymentMethod: React.FunctionComponent = () => {
+  //   const route = useRoute();
+  const bannerRef = useRef<any>();
+  const navigation = useNavigation<any>();
+
+  //   const operation = route.params.operation;
+
+  return (
+    <Fragment>
+      <ScreenComponent>
+        <NavHeader />
+        <View style={styles.container}>
+          <Text style={styles.title}>
+            How do you want to top up your wallet{" "}
+            {/* {operation === "TopUp" ? "top up" : "withdraw from"} your wallet? */}
+          </Text>
+
+          <PaymentMethodButton
+            title="Mpesa"
+            subTitle="Deposit funds using mpesa"
+            image={MPESA}
+            handleAction={() =>
+              navigation.navigate("Add Funds", {
+                // operation: operation,
+              })
+            }
+          />
+          <PaymentMethodButton
+            title="Cash"
+            subTitle="Deposit funds through a cash agent"
+            image={TOKENS}
+            handleAction={() => bannerRef.current?.openBanner()}
+          />
+        </View>
+      </ScreenComponent>
+      <Banner
+        ref={bannerRef}
+        style={{ height: 350 }}
+        content={<BannerContent bannerRef={bannerRef} />}
+      />
+    </Fragment>
+  );
+};
+
+const styles = StyleSheet.create<any>({
+  container: {
+    margin: 30,
+  },
+
+  title: {
+    fontSize: 16,
+    lineHeight: 19,
+    color: "#333333",
+    marginBottom: 65,
+    textAlign: "center",
+    fontFamily: FONTS.sh2.fontFamily,
+  },
+});
+
+const bannerStyles = StyleSheet.create<any>({
+  container: {
+    height: "100%",
+    paddingVertical: 20,
+    alignItems: "center",
+    justifyContent: "space-between",
+  },
+
+  image: {
+    height: 150,
+    maxWidth: SIZES.width * 0.8,
+    resizeMode: "contain",
+    marginBottom: 20,
+  },
+
+  title: {
+    fontSize: 18,
+    lineHeight: 22,
+    color: "#2C2948",
+    textAlign: "center",
+    fontFamily: FONTS.sh2.fontFamily,
+  },
+
+  text: {
+    fontSize: 14,
+    lineHeight: 21,
+    color: COLORS.textPrimary,
+    textAlign: "center",
+    fontFamily: FONTS.h1.fontFamily,
+  },
+
+  buttonContainer: {
+    height: 50,
+    width: 100,
+    justifyContent: "center",
+  },
+
+  button: {
+    fontSize: 14,
+    lineHeight: 23,
+    color: "#4840BB",
```

Comment:
try ...FONTS.h5 instead of re-writing the text formatting styles
and pick the color from the COLORs file